src/Controls/src/Core/Label/Label.iOS.cs

```diff
@@ -38,13 +38,6 @@ public static void MapMaxLines(ILabelHandler handler, Label label)
 
 		static void MapFormatting(ILabelHandler handler, Label label)
```

Comment:
I feel like I wrote this code and overengineered this. It appears there is just 1 usage of the MapFormatting method, and it was already weird. Maybe this should just be merged with the MapText or UpdateText?